view/utils/init-table-sorting-on-client.js

```diff
@@ -34,9 +34,22 @@ module.exports = function (domElement/*, opts */) {
 		type: 'numeric'
 	});
 	var opts = normalizeOptions(arguments[1]);
-	setInterval(function () {
-		var element = window.jQuery(domElement);
-		element.trigger('update');
-		element.tablesorter(opts);
-	}, 500);
+	var additionalOpts = normalizeOptions(arguments[2]);
+
+	if (additionalOpts && additionalOpts.beforeSortStartFn) {
```

Comment:
no. I made this decision because element.trigger('update'); always executes sortStart. Because of that when you open a detailrow in a table it gets sorted after 500ms... Basically we are relying on luck here. To my "defence" I can say that this example luckily worked well on http://ilja9032-els.eregistrations.org/time/per-person/ with production db